### ISSUE TITLE
Nothing in SoundCloud/Following

### ISSUE THREAD

**Issue by THS-on** [mopidy/mopidy-soundcloud#67]
Hello,
when I open the "SoundCloud/Following" folder I get this error message:

``` python
ERROR    SoundCloudBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 19, in _backend_error_handling
    yield
  File "/usr/lib/python2.7/dist-packages/mopidy/core/library.py", line 112, in _browse
    result = backend.library.browse(uri).get()
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/library.py", line 147, in browse
    return self.list_user_follows()
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/library.py", line 89, in list_user_follows
    for (name, user_id) in self.backend.remote.get_followings():
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/soundcloud.py", line 149, in get_followings
    name = playlist.get('username')
AttributeError: 'unicode' object has no attribute 'get'
```

Runnig mopidy on Debian Jessie and Python 2.7.9.

THS-on

_Updated: Formatted stack trace. -jodal_

**Comment by vasyugan** [comment]
Seeing the same here. Uninstalled the mopidy-soundcloud package and installed it from pip, but to no avail.

**Comment by megalithic** [comment]
same issue/same error; using homebrew install (mopidy/mopidy/mopidy-soundcloud)

**Comment by THS-on** [comment]
You can patch your installation with the changes from @kriim and then it works.
Link to patch: https://github.com/kriim/mopidy-soundcloud/commit/6a406f788c1878c6eb523a17ce8c42260efeaa00.patch

**Comment by vasyugan** [comment]
thanks THS-on, that helps (after restarting mopidy)

**Comment by jarpy** [comment]
Also fixed in #83, I think.
